src/components/Settings/Settings.js

```diff
@@ -1,13 +1,72 @@
 import React from "react";
 import PasswordChange from "../PasswordChange/PasswordChange";
+import CancelAccount from "../CancelAccount/CancelAccount";
 import withAuthorization from "../WithAuthorization";
+import { withStyles } from "@material-ui/core/styles";
+import ExpansionPanel from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import Typography from "@material-ui/core/Typography";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import DeleteIcon from "@material-ui/icons/Delete";
+import PermIcon from "@material-ui/icons/PermIdentity";
+import NotificationsIcon from "@material-ui/icons/SpeakerNotes";
 
-const Settings = () =>
-  <div>
-    <h1>Settings</h1>
-    <PasswordChange />
-  </div>
+const styles = theme => ({
+  root: {
+    width: "100%"
+  },
+  heading: {
+    fontSize: theme.typography.pxToRem(15),
+    fontWeight: theme.typography.fontWeightRegular
+  }
+});
 
-const authCondition = (authUser) => Boolean(authUser);
+class Settings extends React.Component {
+  render() {
+    const { classes } = this.props;
 
-export default withAuthorization(authCondition)(Settings);
\ No newline at end of file
+    return (
+      <div className={classes.root}>
+        <Typography variant="headline">Settings</Typography>
+        <br />
+        <br />
+        <ExpansionPanel>
+          <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+            <PermIcon />
+            <Typography className={classes.heading}>Change password</Typography>
+          </ExpansionPanelSummary>
+          <ExpansionPanelDetails>
+            <PasswordChange />
```

Comment:
We can add marginRight: theme.spacing.unit * 3 to the icons in this section.

Reply:
Good idea! Done!